src/Compilers/Core/Portable/Compilation/IExpression.cs

```diff
@@ -841,10 +841,36 @@ public interface IConversionExpression : IHasOperatorMethodExpression
         /// Kind of conversion.
         /// </summary>
         ConversionKind ConversionKind { get; }
+
+        /// <summary>
+        /// Returns true if the conversion is an identity conversion.
+        /// </summary>
+        bool IsIdentity { get; }
+
+        /// <summary>
+        /// Returns true if the conversion is an implicit ('widening' in VB) conversion.
+        /// </summary>
+        bool IsImplicit { get; }
+
         /// <summary>
         /// True if and only if the conversion is indicated explicity by a cast operation in the source code.
         /// </summary>
         bool IsExplicit { get; }
+
+        /// <summary>
+        /// Returns true if the conversion is an implicit numeric conversion or explicit numeric conversion. 
+        /// </summary>
+        bool IsNumeric { get; }
+
+        /// <summary>
+        /// Returns true if the conversion is an implicit nullable conversion or explicit nullable conversion.
+        /// </summary>
+        bool IsNullable { get; }
+
+        /// <summary>
+        /// Returns true if the conversion is an implicit reference conversion or explicit reference conversion.
+        /// </summary>
+        bool IsReference { get; }
```

Comment:
> Perhaps make it a combination of bit flags.

I would be very wary of doing this through bitflags.  That's not how we've traditionally exposed any compiler or IDE information.  BitFlags seem great as an internal storage decision.  Not as something people have to comprehend on their own.

> Alternatively, we can add one new property that returns a different new interface with all these properties

That's potentially doable.  @JohnHamby What do you think?